### ISSUE TITLE
Unable to create selenium session

### ISSUE THREAD

**Issue by RayUno** [fhoeben/hsac-fitnesse-fixtures#240]
Hi guys,

When I try to execute the example tests in hsac-fitnesse-fixtures-3.29.1-standalone I get the following failure:

_browser test nl.hsac.fitnesse.fixture.slim.StopTestException: Unable to create selenium session using: nl.hsac.fitnesse.fixture.util.selenium.driverfactory.LocalDriverFactory@7bd7d6d6_
...
_Caused by: org.openqa.selenium.SessionNotCreatedException: session not created
from disconnected: unable to connect to renderer_

I am quite new to this, so I am having a hard time trying to solve this. What I am able to figure out is that the test is opening the browser (I can see it starting on my desktop), but closing it before running tests. When a test has 3 steps, the browser opens and closes 3 times. 

After using google, most suggestions where updating chromedriver, so I re-downloaded the latest version into the project, but without any luck. Any more suggestions I could try?

Thanks!
Ray

**Comment by fhoeben** [comment]
Can you elaborate on your environment? What OS are you on, which version of Chrome, which version of Java? Did you start from the 'standalone.zip' or did you clone/copy the fixture project, or did you clone/copy the 'sample project'? Did you start via Maven, of directly with java from the command line? Did you just upgrade your environment or make a fresh start?

I assume you get the message when starting the tests from a locally running wiki (e.g. http://localhost:9090/HsacExamples.SlimTests.BrowserTests.StoryboardTest). Does this occur for all tests in the BrowserTests suite)?

Usually errors like this are indeed caused by chromedriver and chrome not playing nicely together. I bundled the latest chromedriver I was aware of (2.43) when I made 3.29.1. Sometimes problems can be caused by older versions of chromedriver already/still being present in the `wiki/webdrivers` directory, or even still as running processes (visible in 'Task Manager' on Windows or with 'ps' on Unix based systems)

**Comment by RayUno** [comment]
I think I have all the latest versions. Updated chrome, redownloaded chromedriver as I said. Below output is from the summary in fitnesse, showing my system and the versions

_(Session info: chrome=70.0.3538.77)
(Driver info: chromedriver=2.43.600229 (3fae4d0cda5334b4f533bede5a4787f7b832d052),platform=Mac OS X 10.14.1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 936 milliseconds
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'MacBook-Pro-7.local', ip: '******', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.1', java.version: '1.8.0_181'_

I downloaded the zip, extracted in a different folder. I started Fitnesse using the command line, running java -jar (even tried on different ports like 8080 and 9191). I did nothing in fitnesse besides running the wiki-tests. Test running headless chrome and 'normal' chrome give the same results. A collegue tried it on his machine. Comparison only showed difference in chromedriver (2.43 vs 2.42).
